test/controllers/employment/EmploymentDetailsControllerSpec.scala

```diff
@@ -18,20 +18,18 @@ package controllers.employment
 
 import common.SessionValues
 import config.{MockAuditService, MockIncomeTaxUserDataService}
-import config.MockIncomeTaxUserDataService
-import play.api.mvc.Results.Redirect
-import models.employment.{AllEmploymentData, EmploymentData, EmploymentSource, Pay}
+import models.employment.EmploymentData
 import play.api.http.Status._
 import play.api.mvc.Result
-import play.api.mvc.Results.Ok
+import play.api.mvc.Results.{Ok, Redirect}
 import utils.UnitTestWithApp
-import views.html.employment.EmploymentDetailsView
+import views.html.employment.CheckEmploymentDetailsView
 
 import scala.concurrent.Future
 
 class EmploymentDetailsControllerSpec extends UnitTestWithApp with MockIncomeTaxUserDataService with MockAuditService{
```

Comment:
should this test be changed to 'CheckEmploymentDetailsControllerSpec' to match the view and integration test?